tutorials/platform/android/android_plugin.rst

```diff
@@ -71,6 +71,14 @@ to set up your own Godot Android plugin project.
 To provide further understanding, here is a break-down of the steps used to create the project template:
 
 1. Create an Android library module using `these instructions <https://developer.android.com/studio/projects/android-library>`_
+1.a. remove version name and version code from the build.gradle.kts as those attributes are not necessary for library modules 
+    android{
+        defaultConfig{ }
+    }
```

Comment:
This can be omitted and left to the developer's discretion. For example, I tend to use those attributes to track versioning for the libraries I write.